### PR TITLE
'Leather' theme

### PR DESCRIPTION
**Pull request by ryujinscales**
image
above : flip test colors and colorful mode off

image
above :  flip test colors on

image
above ; flip test colors and colorful mode on

image
above : colorful mode on

Hope it get's approved !
Signing off, ryujinscales

### CODE REVIEW

**Comment by rizwanmustafa** [comment]
Remove the greek fire theme cz it has already been rejected.

File: static/themes/greek_fire.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:root {
```
Comment (rizwanmustafa) [inline static/themes/greek_fire.css:1]: Remove this file.

File: static/themes/_list.json [hunk 2]
```diff
@@ -699,4 +704,14 @@
     "bgColor": "linear-gradient(to right, #002c39, #015061, #007a7f, #00e980, #00ffbb)",
     "mainColor": "#ffffff"
   }
+  ,{
```
Comment (rizwanmustafa) [inline static/themes/_list.json:707]: Remove the greek_fire entry

File: static/themes/ryujinscales.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+:root {
```
Comment (rizwanmustafa) [inline static/themes/ryujinscales.css:1]: This file is the same as the greek theme, so remove it.

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]

File: static/themes/ryujinscales.css (hunk 3 above)
Comment (ryujinscales) [reply static/themes/ryujinscales.css:1]: bruh the ryujinscales theme has been approved

File: static/themes/_list.json (hunk 2 above)
Comment (ryujinscales) [reply static/themes/_list.json:707]: sure

File: static/themes/ryujinscales.css (hunk 3 above)
Comment (rizwanmustafa) [reply static/themes/ryujinscales.css:1]: It is literally the same as the greek fire theme. If the greek fire theme was rejected, how can this be approved. And if this was approved, can you provide me the pr link.

**Comment by ryujinscales** [comment]
Pull_request by ryujinscales · Pull Request #2368 · Miodec/monkeytype
(github.com) <https://github.com/Miodec/monkeytype/pull/2368>

File: static/themes/ryujinscales.css (hunk 3 above)
Comment (ryujinscales) [reply static/themes/ryujinscales.css:1]: https://github.com/Miodec/monkeytype/pull/2368

Comment (ryujinscales) [reply static/themes/ryujinscales.css:1]: see for urself
then consult me

Comment (rizwanmustafa) [reply static/themes/ryujinscales.css:1]: Bro, if the theme was already merged, why are you adding it again. Remove this file as the ryujinscales theme has already been added.

Comment (rizwanmustafa) [reply static/themes/ryujinscales.css:1]: And from what I believe, this file is the culprit of merge conflicts.

Comment (ryujinscales) [reply static/themes/ryujinscales.css:1]: > The ryujinscales theme has already been added and merged into Monkeytype, so please remove this file.

which file
send the link
im not sure

Comment (ryujinscales) [reply static/themes/ryujinscales.css:1]: r u sure it wont delete the theme

**Comment by rizwanmustafa** [comment]
@ryujinscales  https://github.com/Miodec/monkeytype/blob/master/static/themes/ryujinscales.css

**Comment by ryujinscales** [comment]
> @ryujinscales https://github.com/Miodec/monkeytype/blob/master/static/themes/ryujinscales.css

#2397

**Comment by rizwanmustafa** [comment]
What I meant by sending the file link was that the ryujinsacales theme is already added so delete the file in this pr and not the project as a whole.

File: static/themes/ryujinscales.css (hunk 3 above)
Comment (Miodec) [reply static/themes/ryujinscales.css:1]: Since the theme has been already merged in it will not delete it

**Comment by Miodec** [comment]
I recommend you fetch all changes from upstream and merge to get rid of the conflicts.

**Comment by Miodec** [comment]
As for the theme itself, please include some screenshots when not typing too.

Also i feel like the background color is too dark. I feel like leather is usually bit more lighter than this.

**Comment by ryujinscales** [comment]
> I recommend you fetch all changes from upstream and merge to get rid of the conflicts.

and how exactly do i do that

**Comment by ryujinscales** [comment]
> As for the theme itself, please include some screenshots when not typing too.
> 
> Also i feel like the background color is too dark. I feel like leather is usually bit more lighter than this.

i updated it

**Comment by ryujinscales** [comment]
> > As for the theme itself, please include some screenshots when not typing too.
> > Also i feel like the background color is too dark. I feel like leather is usually bit more lighter than this.
> 
> i updated it

updated bgcolor

image

**Comment by Miodec** [comment]
How about this instead
image

**Pull request merged**